src/tree/size-constants.js

```diff
@@ -0,0 +1,17 @@
+const buttonMargin = 12;
+const buttonHeight = 24;
+
+const constants = {
+  cardWidth: 152,
+  cardHeight: 64,
+  widthMargin: 32,
+  heightMargin: buttonHeight + buttonMargin * 3,
+  leafMargin: 16,
+  buttonMargin,
+  buttonWidth: 48, // TODO: might need to be dynamic
```

Comment:
you could probably just skip this, and instead set a max-width in css, just to avoid strange large cases.

Reply:
Exactly, to center them becomes a problem if the css sets the width dynamically. I've tried and three digits fit, so we'll never need more than that if we have a sanity limit.